sys/shell/commands/sc_gnrc_ipv6_nib.c

```diff
@@ -57,16 +57,16 @@ static void _usage_nib_neigh(char **argv)
     printf("usage: %s %s [show|add|del|help]\n", argv[0], argv[1]);
     printf("       %s %s add <iface> <ipv6 addr> [<l2 addr>]\n", argv[0], argv[1]);
     printf("       %s %s del <ipv6 addr>\n", argv[0], argv[1]);
-    printf("       %s %s show <ipv6 addr>\n", argv[0], argv[1]);
+    printf("       %s %s show [ipv6 addr]\n", argv[0], argv[1]);
 }
 
 static void _usage_nib_prefix(char **argv)
 {
     printf("usage: %s %s [show|add|del|help]\n", argv[0], argv[1]);
-    printf("       %s %s add <iface> <prefix>[/<prefix_len>] [<valid in ms>] [<pref in ms>]\n",
+    printf("       %s %s add <iface> <prefix>[/<prefix_len>] [<valid in sec>] [<pref in sec>]\n",
```

Comment:
I don't know about this. The lifetime *has* millisecond precision, so why not use it. Also this would require, to check, that `ltime < (UINT32_MAX / MS_PER_SEC)` which makes the shell command more complicated than it needs to be IMHO. 

I don't see it as an inconsistency either, that's why the times are documented here.

Reply:
> I don't know about this. The lifetime has millisecond precision, so why not use it.

Because IMHO it is overkill for a shell application, when I really require my prefix to have millisecond accuracy I would use a programmatical way to add the prefix as the inaccuracy of the shell is far too large to achieve that millesecond accuracy.
With the millisecond accuracy, the whole process of manually adding a prefix becomes more error prone due to the large numbers that are less readable.

> that ltime < (UINT32_MAX / MS_PER_SEC) which makes the shell command more complicated than it needs to be IMHO.

Fair enough.

> I don't see it as an inconsistency either, that's why the times are documented here.

Okay, let me rephrase that: Why does the `prefix add` command use millisecond for the input and is the `prefix show` using seconds for the output?

Reply:
Mhhhhh... I think I am convinced.